Makefile

```diff
@@ -123,24 +128,20 @@ coverage:
 	@go tool cover -func coverage.txt
 
 .PHONY: lint
-lint: $(GOLANGCI_LINT) $(LICENSER) $(GORELEASER) .golangci.yml .goreleaser.yaml ## Run the linters
+lint: .golangci.yml .goreleaser.yaml ## Run the linters
 	@echo "--- lint ---"
-	@$(LICENSER) verify -r .
-	@$(GOLANGCI_LINT) run --timeout 5m --config .golangci.yml ./...
-	@$(GORELEASER) check -q
+	@go run $(licenser) verify -r .
+	@go run $(golangci_lint) run --timeout 5m --config .golangci.yml ./...
+	@go run $(goreleaser) check -q
 
 # The goimports tool does not arrange imports in 3 blocks if there are already more than three blocks.
 # To avoid that, before running it, we collapse all imports in one block, then run the formatter.
 .PHONY: format
-format: $(GOIMPORTS) ## Format all Go code
+format: ## Format all Go code
 	@echo "--- format ---"
-	@$(LICENSER) apply -r "Tetrate"
+	@go run $(licenser) apply -r "Tetrate"
 	@find . -type f -name '*.go' | xargs gofmt -s -w
-	@for f in `find . -name '*.go'`; do \
-	    awk '/^import \($$/,/^\)$$/{if($$0=="")next}{print}' $$f > /tmp/fmt; \
-	    mv /tmp/fmt $$f; \
-	    $(GOIMPORTS) -w -local github.com/tetratelabs/func-e $$f; \
```

Comment:
doing this in a loop is as time consuming as trying to parse wtf ^^ was trying to do